### PR TITLE
Remove ncBrowse

### PR DESCRIPTION
**Pull request by ZMAlt**

### CODE REVIEW

File: source/module/mgd77convert.rst [hunk 1]
```diff
@@ -128,14 +128,12 @@ mgd77convert
 其他工具
 --------
 
-MGD77+ NetCDF 文件符合 CF 1.0 和 COARDS 标准，可以使用 ncBrowse 和 ncView 等
+MGD77+ NetCDF 文件符合 CF 1.0 和 COARDS 标准，可以使用 NetCDF 库自带的 ncdump 命令或 ncView 等
```
Comment (seisman) [inline source/module/mgd77convert.rst:131]:
```suggestion
MGD77+ NetCDF 文件符合 CF 1.0 和 COARDS 标准，可以使用 NetCDF 库自带的 ``ncdump`` 命令或 ncView 等
```

File: source/grid/viewer.rst [hunk 2]
```diff
@@ -1,11 +1,16 @@
 查看 netCDF 文件
 ================
 
-某些软件可以直接用于查看 netCDF 文件的内容：
+使用 NetCDF 库自带的 ncdump 命令可以很方便地查看 NetCDF 文件内容。
```
Comment (seisman) [inline source/grid/viewer.rst:4]:
```suggestion
使用 NetCDF 库自带的 ``ncdump`` 命令可以很方便地查看 NetCDF 文件内容。
```

File: source/module/mgd77convert.rst (hunk 1 above)
Comment (seisman) [inline source/module/mgd77convert.rst:131]:
```suggestion
MGD77+ NetCDF 文件符合 CF 1.0 和 COARDS 标准，可以使用 NetCDF 库自带的 ``ncdump`` 命令或 ncView 等
```

Comment (ZMAlt) [reply source/module/mgd77convert.rst:131]: duplication of the previous one.

File: source/grid/viewer.rst [hunk 3]
```diff
@@ -1,11 +1,16 @@
 查看 netCDF 文件
 ================
 
-某些软件可以直接用于查看 netCDF 文件的内容：
+使用 NetCDF 库自带的 ``ncdump`` 命令可以很方便地查看 NetCDF 文件内容。
+由于 NetCDF 库为 GMT 必需依赖库，因此安装好 GMT 后，通常已成功安装
+该库及 ncdump 等命令，如果无法直接执行该命令，可尝试通过添加环境
```
Comment (ZMAlt) [inline source/grid/viewer.rst:6]:
```suggestion
该库及 ``ncdump`` 等命令，如果无法直接执行该命令，可尝试通过添加环境
```

**Pull request merged**